SwanProjects/bin/swan_bash

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash -i
+#Author Omar.Zapata@cern.ch 2021
+###
+# This script allows to start a bash interpreter inside a project environment.
+# This is called from the extension SwanTerminal and it requires like a parameter the project name to
```

Comment:
This is called from the extension SwanTerminal and it requires the project name as argument. The project name
is used to build the path to the project with $HOME/SWAN_projects/$PROJECT, so that the project environment
can be loaded.

Reply:
BTW swan_bash now requires the project path and the stacks path, then I changed the documentation for the new parameters.